wiiu/vino/vino.ts

```diff
@@ -279,16 +366,16 @@ export default interface VinoAPI {
 	navi_setAreaOverSearch: (unk: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk - Unknown
+	 * Changes navigation moving method
+	 * @param method - -1 disables navigation, 1 is default
```

Comment:
You've changed the description of the method but the values don't line up? Either that or your description of the values is poorly worded

Also if a parameter is limited to certain values, it should be an enum